services/QuillLMS/app/services/stripe_integration/webhooks/checkout_session_expired_event_handler.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module StripeIntegration
+  module Webhooks
+    class CheckoutSessionExpiredEventHandler < EventHandler
```

Comment:
This is identical to the handler above, is there a reason to not reuse it?

Reply:
I thought about it, but since it's only two examples, I'll leave as-is for now.